app/(aux)/settings.js

```diff
@@ -671,20 +1242,29 @@ export default function Page() {
 				ref={bottomSheetModalRef}
 				index={0}
 				snapPoints={
```

Comment:
This multi-nested ternary is not really the best idea. you should create a stateful variable 
`const [snapPoints, setSnapPoints] = useState([]);` 
and update this when you toggle the bottom sheet modal(my proposed parameterized function accounts for this). 

after that simply replace this entire block with 
`snapPoints={SnapPoints(snapPoints)}` 
(after importing SnapPoints from the helper file)